website/static/js/markdown.js

```diff
@@ -31,7 +31,7 @@ var bootstrapTable = function(md) {
 var oldMarkdownList = function(md) {
     md.block.ruler.after('hr', 'pyMarkdownList', pymarkdownList);
 };
-var mfrURL = window.contextVars.mfrURL;
+var mfrURL = window.contextVars.node.urls.mfr;
```

Comment:
@sloria, question about mfr_urls and wiki rendering.  This change will use the mfr url stored on the region.  But what if the guid I am trying to embed is for a file stored on Google Drive, for example?  Shouldn't I use the default mfr_url, as is being done elsewhere in this PR with `get_mfr_url`?

 However, I don't know how to anticipate what provider the guid I am trying to embed is from.  What happens if the mfr_url I use is from the node's region, but the file is not in osfstorage?

Reply:
Hm, do we allow embedding non-OSF files?

Reply:
well a github file worked!

Reply:
Yeah, MFR should be able to render files for addons, regardless of the region the MFR instance is located. So I think it should Just Work.